conf/tikv.yml

```diff
@@ -364,71 +272,13 @@ rocksdb:
   ## Use O_DIRECT for both reads and writes in background flush and compactions.
   # use-direct-io-for-flush-and-compaction: false
 
-  ## Limit the disk IO of compaction and flush.
-  ## Compaction and flush can cause terrible spikes if they exceed a certain threshold. Consider
-  ## setting this to 50% ~ 80% of the disk throughput for a more stable result. However, in heavy
-  ## write workload, limiting compaction and flush speed can cause write stalls too.
-  ## 1. rate-bytes-per-sec is the only parameter you want to set most of the time. It controls the
-  ##    total write rate of compaction and flush in bytes per second. Currently, RocksDB does not
-  ##    enforce rate limit for anything other than flush and compaction, e.g. write to WAL.
-  ## 2. rate-limiter-mode indicates which types of operations count against the limit.
-  ##    1 : ReadOnly
-  ##    2 : WriteOnly
-  ##    3 : AllIo
-  ## 3. auto_tuned enables dynamic adjustment of rate limit within the range
-  ##    [rate_bytes_per_sec / 20, rate_bytes_per_sec], according to the recent demand for background I/O.
-  # rate-bytes-per-sec: 0
-  # rate-limiter-mode: 2
-  # auto-tuned: false
-
-  ## Enable or disable the pipelined write.
-  # enable-pipelined-write: true
-
   ## Allows OS to incrementally sync files to disk while they are being written, asynchronously,
   ## in the background.
   # bytes-per-sync: "1MB"
 
   ## Allows OS to incrementally sync WAL to disk while it is being written.
   # wal-bytes-per-sync: "512KB"
 
-  ## Specify the maximal size of the RocksDB info log file.
-  ## If the log file is larger than this config, a new info log file will be created.
-  ## If it is set to 0, all logs will be written to one log file.
-  # info-log-max-size: "1GB"
-
-  ## Time for the RocksDB info log file to roll (in seconds).
-  ## If the log file has been active longer than this config, it will be rolled.
-  ## If it is set to 0, rolling will be disabled.
-  # info-log-roll-time: "0"
-
-  ## Maximal RocksDB info log files to be kept.
-  # info-log-keep-log-file-num: 10
-
-  ## Specifies the RocksDB info log directory.
-  ## If it is empty, the log files will be in the same directory as data.
-  ## If it is not empty, the log files will be in the specified directory, and the DB data directory's
-  ## absolute path will be used as the log file name's prefix.
-  # info-log-dir: ""
-
-  ## Options for "Titan"
-  titan:
```

Comment:
why remove this? 🤣can you add it back?

Reply:
All config items which been reduced have been added back.

Reply:
which release version

Reply:
master、3.1 and 3.0

Reply:
didn't see it in v3.0.7 and master

Reply:
Update, added back in 3.0 and 3.1, master did not add. The released version can't be added back.

Reply:
Can we also revert the change for titan in master?